src/playback.jl

```diff
@@ -0,0 +1,45 @@
+
+"""
+    RandomPolicy{RNG<:AbstractRNG, P<:Union{POMDP,MDP}, U<:Updater}
```

Comment:
typo, I think it should be PlaybackPolicy

Reply:
Yes, that is a typo, thanks!